src/DETHRACE/CMakeLists.txt

```diff
@@ -195,8 +200,38 @@ endif()
 
 if (DETHRACE_INSTALL)
     install(TARGETS dethrace
-        RUNTIME DESTINATION "."
+        BUNDLE DESTINATION . COMPONENT Runtime
+        RUNTIME DESTINATION . COMPONENT Runtime
     )
+
+    if (APPLE)
+        set_target_properties(dethrace PROPERTIES
+            BUNDLE True
+            MACOSX_BUNDLE True
+            MACOSX_BUNDLE_DISPLAY_NAME "Deth Race"
+            OUTPUT_NAME "Deth Race"
+            MACOSX_BUNDLE_BUNDLE_VERSION ${DETHRACE_VERSION}
+            MACOSX_BUNDLE_SHORT_VERSION_STRING ${DETHRACE_VERSION}
+            MACOSX_BUNDLE_ICON_FILE dethrace.icns
+        )
+        set_source_files_properties(${CMAKE_SOURCE_DIR}/packaging/macos/dethrace.icns PROPERTIES
+            MACOSX_PACKAGE_LOCATION "Resources"
+        )
+        target_sources(dethrace_obj PRIVATE
+            ${CMAKE_SOURCE_DIR}/packaging/macos/dethrace.icns
+        )
```

Comment:
Do we need an option to build DethRace on macos *not* as a bundle, as an ordinary cli executable?

Reply:
without `-DDETHRACE_INSTALL`, and ordinary executable is created

Reply:
True :)
That seems good enough for me.